src/instructlab/eval/mmlu.py

```diff
@@ -167,36 +207,21 @@ def run(self) -> tuple:
         return overall_score, individual_scores
 
 
-class MMLUBranchEvaluator(Evaluator):
+class MMLUBranchEvaluator(AbstractMMLUEvaluator):
     """
-    Child class of an Evaluator for Massive Multitask Language Understanding Branch (MMLUBranch)
+    Evaluator for Massive Multitask Language Understanding Branch (MMLUBranch)
 
     Attributes:
-        model_path  absolute path to or name of a huggingface model
-        sdg_path    path where the <TASK_NAME>.jsonl and <TASK_NAME>_task.yaml files for the branches being evaluated are stored
-        tasks       group name that is shared by all the MMLUBranch tasks
-        few_shots   number of examples
-        batch_size  number of GPUs
+        model_path      absolute path to or name of a huggingface model
+        sdg_path        path where the <TASK_NAME>.jsonl and <TASK_NAME>_task.yaml files for the branches being evaluated are stored
+        tasks           group name that is shared by all the MMLUBranch tasks
+        model_dtype     dtype of model when served
+        few_shots       number of examples
+        batch_size      number of GPUs
     """
 
     name = "mmlu_branch"
```

Comment:
what's this for? I don't see it referenced elsewhere

Reply:
https://github.com/instructlab/eval/commit/f6ac6566c2710a0f06bb390447d4c4968c5dec5a